vscode-dotnet-runtime-extension/CHANGELOG.md

```diff
@@ -7,8 +7,15 @@ and this project adheres to [Semantic Versioning].
 
 ## [Unreleased]
 
-## [2.0.4]
+## [2.0.4] - 2024-05-03
+Adds the setting `sharedExistingDotnetPath` to use an existing install for all extensions, instead of having to set the setting for every individual extension.
 
+UX improvements for C# DevKit and the Global SDK API.
+Bug fixes for the recommended version API when the user is offline or on an unsupported distro.
+
+Improved error logging for diagnosing issues that users report.
+Improved error messages.
+Updates to dependencies and a simplified dependency chain.
```

Comment:
Are you intending these elements to be on separate lines? Because that's not how markdown works. It ends up rendering like this:
![image](https://github.com/dotnet/vscode-dotnet-runtime/assets/17788297/3871f6e4-65a8-44b9-ae89-df284f8c48bf)

There are multiple ways if you want the text to show up exactly as you typed it, but this would do it (`\` at the end of the line):

UX improvements for C# DevKit and the Global SDK API.\
Bug fixes for the recommended version API when the user is offline or on an unsupported distro.

Improved error logging for diagnosing issues that users report.\
Improved error messages.\
Updates to dependencies and a simplified dependency chain.

Reply:
Thank you, I will incorporate this feedback.